### PR TITLE
fixes #44; added Binarylang and Moe

### LINKED ISSUE #44: Binarylang deserves place in this list
I've been using @sealmove [Binarylang](https://github.com/sealmove/binarylang)  extensively for my projects, and it really makes Nim shine compared to what would require to do the same in other system languages.

### PR DESCRIPTION
**Pull request by ArikRahman**
fixes #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -649,7 +650,8 @@ Nim provides unique features for seamless and transparent interoperability with
 - [pararules](https://github.com/paranim/pararules) - A RETE-based rules engine made for games.
 
 
-## Development Tools
+### Development Tools
+- [moe](https://github.com/fox0430/moe) - A vim-like editor made with Nim, also supports C, Rust, Javascript, etc.
```
Comment (ringabout) [inline README.md:654]: It should be put under `Editor Integration` section.

Comment (ArikRahman) [reply README.md:654]: Gotcha, I'll change to that asap.

**Comment by ArikRahman** [comment]
OK should be good now 👍

**Review by ringabout — APPROVED** [review]
Nice work!

**Pull request merged**